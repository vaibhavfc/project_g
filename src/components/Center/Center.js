import React from 'react';
import './Center.scss';

const Center = (props) => (
  <div className="center">
    {props.children}
  </div>
);

export default Center;
