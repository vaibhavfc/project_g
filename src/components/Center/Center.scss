.center {
  display: flex;
  justify-content: center;
  align-items: center;
  height: min-content;
  padding: 2rem;
}
