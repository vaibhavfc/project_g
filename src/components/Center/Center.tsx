import React from 'react';
import './Center.scss';

const Center = (props: any) => {
  const { children } = props;
  return (
    <div className="center">
      {children}
    </div>
  );
};

export default Center;
