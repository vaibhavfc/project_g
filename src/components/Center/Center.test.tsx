import { render } from '@testing-library/react';

import renderer from 'react-test-renderer';

import Center from './Center';

describe('test', () => {
  it('Renders Center Compoents', () => {
    render(<Center>test</Center>);
  });

  it('Matches Snapshot of Center', () => {
    const tree = renderer.create(<Center> TEST</Center>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
