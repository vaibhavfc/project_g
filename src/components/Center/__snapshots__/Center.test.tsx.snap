// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Matches Snapshot of Center 1`] = `
<div
  className="center"
>
   TEST
</div>
`;
